lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -71,13 +73,19 @@ class _MySpaceWidgetState extends State<MySpaceWidget> {
   Widget build(BuildContext context) {
     return StoreConnector(
         builder: (BuildContext context, MySpaceViewModel viewModel) => Scaffold(
+              key: _scaffoldKey,
               appBar: AppBar(
+                automaticallyImplyLeading: false,
```

Comment:
sorry, why do we need `automaticallyImplyLeading` set to `false`?

Reply:
It removes the based hamburger icon so I can replace it with LinShare one